src/resolver.js

```diff
@@ -50,4 +55,40 @@ export default {
 
     yield createElements(routeMatches, fetchedComponents, fetchedData);
   },
+
+  /**
+   * Generate route data according to their getters, respecting the order of
+   * promises per the `defer` flag on routes.
+   */
+  getData(match, routeMatches) {
```

Comment:
Moved this here because it's a property of the default resolver, and it's not relevant to other resolvers

Reply:
Yep, makes sense